sequencer/src/block/payload.rs

```diff
@@ -96,6 +96,16 @@ impl<TableWord: TableWordTraits> Payload<TableWord> {
         0..self.ns_table.len()
     }
 
+    /// Returns the flat bytes for namespace `ns_id`.
+    pub fn namespace(&self, ns_id: NamespaceId) -> Option<&[u8]> {
```

Comment:
I feel like this should return `Vec<Transaction>` and do the parsing for you

Reply:
done in 77c3e717cb0bcb2f80e6c25d3aa7158e64c07f4c